artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -3160,6 +3160,14 @@
                </xsd:annotation>
             </xsd:element>
 
+            <xsd:element name="auto-delete-created-queues" type="xsd:boolean" default="true" maxOccurs="1" minOccurs="0">
```

Comment:
@michaelandrepearce I had a question from @michalxo on this...

Wan't the default here supposed to be "false" ?